protos/local/temporal/sdk/core/workflow_commands/workflow_commands.proto

```diff
@@ -181,17 +183,17 @@ message ContinueAsNewWorkflowExecution {
     string task_queue = 2;
     /// Inputs to the workflow code. Should be specified. Will not re-use old arguments, as that
     /// typically wouldn't make any sense.
-    repeated common.Payload arguments = 3;
+    repeated temporal.api.common.v1.Payload arguments = 3;
     /// Timeout for a single run of the new workflow.
     google.protobuf.Duration workflow_run_timeout = 4;
     /// Timeout of a single workflow task.
     google.protobuf.Duration workflow_task_timeout = 5;
     /// Memo fields
-    map<string, common.Payload> memo = 6;
+    map<string, temporal.api.common.v1.Payload> memo = 6;
     /// Header fields
-    map<string, common.Payload> headers = 7;
+    map<string, temporal.api.common.v1.Payload> headers = 7;
     /// Search attributes
-    map<string, common.Payload> search_attributes = 8;
+    map<string, temporal.api.common.v1.Payload> search_attributes = 8;
```

Comment:
Like `coresdk.workflow_activation.Start` does, this could be `temporal.api.common.v1.SearchAttributes`